tests/e2e/scripts/keadm_deprecated_e2e.sh

```diff
@@ -95,9 +96,8 @@ function run_test() {
   if [[ $GINKGO_TESTING_RESULT != 0 ]]; then
     echo "Integration suite has failures, Please check !!"
     echo "edgecore logs are as below"
-    journalctl -u edgecore.service -xe > /var/log/kubeedge/edgecore.log
     set -x
-    cat /var/log/kubeedge/edgecore.log
+    journalctl -u edgecore.service --no-pager
```

Comment:
why update this? we need the log file in order to facilitate the positioning of the problem

Reply:
this can output all edgecore logs